src/index.cpp

```diff
@@ -1758,6 +1697,7 @@ void Index<T, TagT, LabelT>::build(const std::string &data_file, const size_t nu
 template <typename T, typename TagT, typename LabelT>
 std::unordered_map<std::string, LabelT> Index<T, TagT, LabelT>::load_label_map(const std::string &labels_map_file)
```

Comment:
index need not have a load_label_map itself.. all label related loads should occur in filter_store->load()